### PR TITLE
update documentation-usability.md/issue-label-inclusivity.md/documentation-discoverability.md

### PR DESCRIPTION
**Pull request by sanglide**
Signed-off-by: 171250022@smail.nju.edu.cn <171250022@smail.nju.edu.cn>

update the following files:

    https://github.com/chaoss/wg-diversity-inclusion/blob/master/focus-areas/project-and-community/documentation-usability.md
    https://github.com/chaoss/wg-diversity-inclusion/blob/master/focus-areas/project-and-community/issue-label-inclusivity.md
    https://github.com/chaoss/wg-dei/blob/main/focus-areas/project-and-community/documentation-discoverability.md

### CODE REVIEW

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md [hunk 1]
```diff
@@ -5,20 +5,36 @@
 文档的可发现性至关重要，因为它在开源项目的包容性中扮演着重要角色。如果人们不能轻松地找到并浏览文档主体，或者找到他们所寻找的答案，他们就不太可能使用或为项目作出贡献，从而降低了项目吸引不同参与者的能力。文档可发现性的目标是确保可搜索性和可见性，以及提高不同受众的包容性。
 
 ## 目标
--**可见性** — 文件很容易找到和参考。
--**可搜索性** — 用户可以根据自己的能力轻松地浏览文档，并且可以通过搜索找到自己需要的相关信息。
--**发现方法** — 确定人们发现文档的方式。
--**包容性** - 知识在项目贡献者之间更公平地分享，新的贡献者更容易参与。
+<<<<<<< HEAD:chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md
```
Comment (eyehwan) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:8]: There are some conflicts that need to be fixed. :)

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md [hunk 2]
```diff
@@ -5,20 +5,36 @@
...
 
-__健康指标的使用和传播可能导致隐私侵犯。组织可能会面临风险。这些风险可能来自于遵守欧盟的 GDPR、美国的州法律或其它法律。此外，GitHub 和 GitLab 等数据提供商的服务条款也可能带来合同风险。请务必检查度量指标的使用是否存在风险和潜在的数据伦理问题。请参阅[CHAOSS 数据伦理文件](https://github.com/chaoss/metrics/tree/main/resources)以获得更多指导。__
+__健康指标的使用和传播可能导致隐私侵犯。组织可能会面临风险。这些风险可能来自于遵守欧盟的 GDPR、美国的州法律或其它法律。此外，GitHub 和 GitLab 等数据提供商的服务条款也可能带来合同风险。请务必检查度量指标的使用是否存在风险和潜在的数据伦理问题。请参阅[CHAOSS 数据伦理文件](https://github.com/chaoss/metrics/tree/main/resources) 以获得更多指导。__
 
 ### 数据收集策略
 
 
-调查项目成员
-李克特量表(很容易找到——不可能找到)关于文档的可发现性，比如:
+<<<<<<< HEAD:chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md
```
Comment (eyehwan) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:27]: There are some conflicts that need to be fixed. :)

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md [hunk 3]
```diff
@@ -39,19 +55,26 @@ __健康指标的使用和传播可能导致隐私侵犯。组织可能会面临
...
-- 要求文档的用户写一个**摩擦日志**，描述他们在文档中遇到的问题。这为文档编辑提供了具体的使用案例，以了解如何为特定用户改进文档。
+与文件的预期用户进行预览文档。
+
+观察他们如何交互和使用文档，以及他们在哪里卡住了。这可以是一个视频会议会话，文档的用户在其中共享他们的屏幕。 
+ 
+要求文档的用户写一个[**摩擦日志**](https://devrel.net/developer-experience/an-introduction-to-friction-logging)，描述他们在文档中遇到的问题。这为文档编辑提供了具体的使用案例，以了解如何为特定用户改进文档。
+<<<<<<< HEAD:chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md
+=======
+
+>>>>>>> 03c4446a6478ba5426b9fa2759bb523b411846b1:chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation_discoverability.md
```
Comment (eyehwan) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:77]: There are some conflicts that need to be fixed. :)

**Review by eyehwan — CHANGES_REQUESTED** [review]
There are some conflicts that need to be fixed. :)

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md [hunk 4]
```diff
@@ -5,20 +5,25 @@
...
 
 ## 实现
 
-__健康指标的使用和传播可能导致隐私侵犯。组织可能会面临风险。这些风险可能来自于遵守欧盟的 GDPR、美国的州法律或其它法律。此外，GitHub 和 GitLab 等数据提供商的服务条款也可能带来合同风险。请务必检查度量指标的使用是否存在风险和潜在的数据伦理问题。请参阅[CHAOSS 数据伦理文件](https://github.com/chaoss/metrics/tree/main/resources)以获得更多指导。__
+__健康指标的使用和传播可能导致隐私侵犯。组织可能会面临风险。这些风险可能来自于遵守欧盟的 GDPR、美国的州法律或其它法律。此外，GitHub 和 GitLab 等数据提供商的服务条款也可能带来合同风险。请务必检查度量指标的使用是否存在风险和潜在的数据伦理问题。请参阅[CHAOSS 数据伦理文件](https://github.com/chaoss/metrics/tree/main/resources) 以获得更多指导。__
 
 ### 数据收集策略
 
+调查项目成员：
+
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:24]: added line breaks

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md [hunk 5]
```diff
@@ -39,19 +44,23 @@ __健康指标的使用和传播可能导致隐私侵犯。组织可能会面临
...
 
 多项选择: 当您开始参与项目时，是否遇到过与文档的可发现性相关的任何挑战(例如，语言障碍或文档结构) ？
-答案选项: 
+答案选项:
 - 没有挑战
-- 一些挑战
-- 几个挑战
-- 许多挑战
+- 少许挑战
+- 有些挑战
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:50]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md (hunk 4 above)
Comment (sanglide) [reply chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:24]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md (hunk 5 above)
Comment (sanglide) [reply chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:50]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md [hunk 6]
```diff
@@ -39,19 +44,23 @@ __健康指标的使用和传播可能导致隐私侵犯。组织可能会面临
...
 
 如果答案不是“没有挑战”，那开放式的后续问题是: 
 - 举个例子，说明你经历了什么挑战，什么时候经历的，挑战如何影响你，如果有的话，你是如何克服挑战的。
 - 开放式的后续问题: 
  - 您对于改进项目文档中关于新贡献者可用的政策、流程或指南有什么建议? 
 
-- 与文件的预期用户进行预览文档。观察他们如何交互和使用文档，以及他们在哪里卡住了。这可以是一个视频会议会话，文档的用户在其中共享他们的屏幕。 
-- 要求文档的用户写一个**摩擦日志**，描述他们在文档中遇到的问题。这为文档编辑提供了具体的使用案例，以了解如何为特定用户改进文档。
+与文件的预期用户进行预览文档。
+
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-discoverability.md:59]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 7]
```diff
@@ -5,38 +5,40 @@
 
 ## 描述
 
-文档可用性确保不同用户都能理解开源项目文档的关键作用。 文档可用性包含结构、内容、术语和可读性等问题，目的是最大限度推动贡献者对项目的理解。
+文档可用性是开源项目文档的一个关键方面，其在于确保不同用户都能理解开源项目的文档。文档可用性涵盖文档的结构、内容、术语和可读性，旨在推动最广大贡献者群体对项目的理解。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:8]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 8]
```diff
@@ -5,38 +5,40 @@
...
 
 
 ## 目标
-* **技术术语** - 文档使用适当程度的技术术语并提供必要的术语解释，确保特定项目的入门贡献者能够理解文档。
+* **技术术语** - 文档使用适当程度的技术术语，并提供必要的术语解释，确保特定项目的入门贡献者能够理解文档。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:12]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 9]
```diff
@@ -5,38 +5,40 @@
...
 * **结构清晰度** - 文档易于阅读和理解。
-* **可读性** - 文档语言应简洁明了，使用常用词和短句，确保非母语者或可能不了解类似缩写惯例或推论模式的人能够理解。
+* **可读性** - 文档语言应简洁明了，使用常用词和短句，以确保非母语者或可能不了解类似缩写惯例或推论模式的人能够理解。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:14]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 10]
```diff
@@ -5,38 +5,40 @@
...
 * **语言包容性** - 文档应避免使用非包容性语言（例如“brogrammer”语言或排斥性/贬义语言）。
 * **语言多元化** - 针对目标受众和不同语言提供通用语言的文档。
-* **时间/注意力多元化** - 文档包容性让人们可以在不同时间读取或设置/运行命令。 还应考虑到需要照顾孩子或具有其他照护职责的人员，这些人员随时可能将注意力转移到屏幕之外。
+* **时间/注意力多元化** - 文档应该有包容性，能够让人们在不同时间读取或设置/运行命令。 还应考虑到需要照顾孩子或具有其他照护职责的人员，这些人员随时可能将注意力转移到屏幕之外。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:17]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 11]
```diff
@@ -5,38 +5,40 @@
...
 ### 数据收集策略
 
-* 采访新人，确定文档如何帮助贡献者 (a) 理解贡献过程，和/或 (b) 完成任务。 样本采访问题：
-  * 描述一下您是如何使用文档了解贡献进程的。
-  * 描述一下您在社区工作中遇到问题时是如何使用文档的。
-  * 描述您是如何使用文件了解和帮助外联工作的。
-  * 您对这些技术术语感到满意吗？ （适用于李克特量表 [1-5] 的调查）
+* 采访新人，确定文档如何帮助贡献者 (a) 理解贡献过程，和/或 (b) 完成任务。 样例采访问题：
+  * 描述一下您使用文档了解贡献过程的经验。
+  * 描述一下您在社区工作中遇到问题时使用文档的经验。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:28]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 12]
```diff
@@ -5,38 +5,40 @@
...
+  * 描述您使用文件了解和帮助外联工作的经验。
+  * 您对这里出现的大量技术术语感到满意吗？ （适用于李克特量表 [1-5] 的调查）
   * 有您不理解的术语或语言吗？
   * 您对改进项目的政策、流程或新贡献者可用的准则有什么建议？
-  * 采访后，社区可追踪回复，记为 `Positive experience`、`Negative experience` 或 `Neutral experience`，逐月报告，了解随时间推移的改进情况。
+  * 采访后，社区可追踪回复，记为 `正面体验`、`负面体验` 或 `中性体验`，并逐月报告这些情况，了解随时间推移的改进情况。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:33]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md [hunk 13]
```diff
@@ -5,38 +5,40 @@
...
 * 提出有关可读性和可扫描性的问题，如：文档是否使用组织结构，如：
   * 标题
   * 文本和代码块
   * 项目符号与段落
   * 锚点
 * 与文档的预期用户进行预审。 观察他们如何使用文档以及在哪里遇到问题。 可以采用视频会议会话，让文档用户共享屏幕。
-* 要求文档用户编写[摩擦日志](https://devrel.net/developer-experience/an-introduction-to-friction-logging)并描述其在文档上遇到的问题。 这为文档编辑提供了具体用例，以了解如何针对具体用户改进文档。
+* 要求文档用户编写[摩擦日志](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
+            并描述其在文档上遇到的问题。 这为文档编辑提供了具体用例，以了解如何针对具体用户改进文档。
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/documentation-usability.md:41]: make some translation changes

File: chinese/wg-diversity-inclusion/focus-areas/project-and-community/issue-label-inclusivity.md [hunk 14]
```diff
@@ -8,7 +8,7 @@
 
 ## 目标
 
-*__新人友好__*：议题包括适当标注的议题（例如，“对新人友好的第一个提议”，“新人友好”），供新贡献者入门
+*__新人友好__*：议题包括适当标注的议题（例如，“对新人友好的第一个议题”，“新人友好”），供新贡献者入门
```
Comment (sanglide) [inline chinese/wg-diversity-inclusion/focus-areas/project-and-community/issue-label-inclusivity.md:11]: make some translation changes

**Comment by eyehwan** [comment]
Looks good to me. Thank you 👍  @sanglide

**Review by eyehwan — APPROVED** [review]

**Pull request merged**